node/full.go

```diff
@@ -396,6 +396,7 @@ func (n *FullNode) Run(ctx context.Context) error {
 		return fmt.Errorf("error while starting data sync service: %w", err)
 	}
 
+	//TODO: once we can identify if we are in based mode we should remove running this for all nodes.
 	go n.reaper.Start(ctx)
```

Comment:
sorry, i might have mislead you here. in the single sequencer mode, we only allow the proposer to relay transactions to the sequencer, not full nodes. so, it is better to encapsulate this under the IsAggregator check. In the base mode, we will have all the nodes have the ability to relay txs as all nodes are run as aggregators (self-proposing full nodes).

Reply:
moved it back, thanks for clarifying